scripts/install_sshnp

```diff
@@ -157,10 +157,10 @@ parse_env() {
 
   # ARCH includes the dot at the end to avoid conflict between arm and arm64
   case "$(uname -m)" in
-    aarch64|arm64) ARCH="arm64.";;
-    x86_64|amd64) ARCH="x64.";;
-    armv7l|arm) ARCH="arm.";;
-    riscv64) ARCH="riscv64.";;
+    aarch64|arm64) ARCH="arm64\.";;
```

Comment:
you could also consider using single quotes

Reply:
This is a bug fix that @cconstab found, I want to match a "." at the end of the string using grep, but grep accepts regex. So I need the "\" in the string to escape the ".".

echo $ARCH outputs "arm64\." in this case.

The one causing issues was "arm." which was matching both "arm64." and "arm."

Reply:
e.g.

```
~/temp 
➜ ls
arm.tgz   arm64.tgz

~/temp 
➜ test_ARCH="arm\."   

~/temp 
➜ ls | grep $test_ARCH
arm.tgz

~/temp 
➜ echo $test_ARCH
arm\.
```

Reply:
> you could also consider using single quotes

See line 173